packages/e2e-tests-playwright/specs/editor/various/new-post.spec.js

```diff
@@ -0,0 +1,120 @@
+/**
+ * Internal dependencies
+ */
+const { test, expect } = require( '../../../config/test' );
+
+test.describe( 'new editor state', () => {
+	test.beforeAll( async ( { testUtils } ) => {
+		await testUtils.activatePlugin(
+			'gutenberg-test-plugin-post-formats-support'
+		);
+	} );
+
+	test.afterAll( async ( { testUtils } ) => {
+		await testUtils.deactivatePlugin(
+			'gutenberg-test-plugin-post-formats-support'
+		);
+	} );
+
+	test( 'should show the New Post page in Gutenberg', async ( {
+		page,
+		testUtils,
+	} ) => {
+		await testUtils.createNewPost();
+
+		expect( page.url() ).toEqual(
```

Comment:
```suggestion
		await expect(page).toHaveURL('**/post-new.php');
```
this automatically retries internally and reduces flakiness by that.